packages/open-payments/src/client/token.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpMethod } from 'openapi'
+import { RouteDeps } from '.'
+import { getASPath, AccessToken } from '../types'
+import { post } from './requests'
+
+export interface TokenRouteDeps extends RouteDeps {
+  client: string
+}
+
+interface RotateRequestArgs {
+  url: string
+}
+
+export interface TokenRoutes {
+  rotate(args: RotateRequestArgs): Promise<AccessToken>
+}
+
+export const createTokenRoutes = (deps: TokenRouteDeps): TokenRoutes => {
+  const rotateTokenValidator =
+    deps.openApi.createResponseValidator<AccessToken>({
+      path: getASPath('/token/{id}'),
+      method: HttpMethod.POST
+    })
+
+  return {
+    rotate: (args: RotateRequestArgs) =>
+      post(
+        deps,
+        {
+          url: args.url
+        },
+        rotateTokenValidator
```

Comment:
I was going to suggest validating the reponse's `access_token.manage` matches `args.url`, but
>The response MUST include an access token management URI, and the value of this URI MAY be different from the URI used by the client instance to make the rotation call.

https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-6-1

Reply:
Yeah, I see. So this doesn't need any update for now I guess.